drillsrs/cmd/study.py

```diff
@@ -35,10 +43,14 @@ def decorate_arg_parser(self, parser: argparse.ArgumentParser) -> None:
         parser.add_argument(
             '-n', type=int, default=10,
             help='set how many flashcards to study')
+        parser.add_argument(
+            '-m', '--mode', type=str, default="direct",
+            help='mode: direct, reversed, mixed')
```

Comment:
Same story about enum and arguments. Since this enum will be shared between the two commands it could be moved to a common place somewhere.